services/text_to_speech/v1-corpus-builder.js

```diff
@@ -47,160 +48,193 @@ module.exports = function (RED) {
     sEndpoint = service.url;
   }
 
-  function executeCreateCustomisation(node, tts, params, msg) {
-    tts.createCustomization(params, function (err, response) {
-      node.status({});
-      if (err) {
-        payloadutils.reportError(node, msg, err);
+  function parseResponseFor(msg, response, field) {
+    if (response && response.result) {
+      if (response.result[field]) {
+        msg[field] = response.result[field];
       } else {
-        msg['customization_id'] = response;
+        msg[field] = response.result;
       }
-      node.send(msg);
+    } else {
+      msg[field] = response;
+    }
+  }
+
+  function executeCreateCustomisation(node, tts, params, msg) {
+    return new Promise(function resolver(resolve, reject) {
+      tts.createVoiceModel(params)
+        .then((response) => {
+          if (response && response.result) {
+            msg['customization_id'] = response.result;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 24 spaces but found 12.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4764899)